twitter_ads/http.py

```diff
@@ -240,13 +242,14 @@ def content_type(self):
     def perform(self):
         """Executes the current TONUpload object."""
 
-        if self._file_size < self._MIN_FILE_SIZE:
+        if self._file_size < self._SINGLE_UPLOAD_MAX:
             resource = "{0}{1}".format(self._DEFAULT_RESOURCE, self.bucket)
             response = self.__upload(resource, open(self._file_path, 'rb').read())
             return response.headers['location']
         else:
             response = self.__init_chunked_upload()
-            chunk_size = int(response.headers['x-ton-min-chunk-size'])
+            min_chunk_size = int(response.headers['x-ton-min-chunk-size'])
+            chunk_size = min_chunk_size * self._DEFAULT_CHUNK_SIZE
```

Comment:
Are we sure that the value of `x-ton-min-chunk-size` won't change? Having the `chunk_size` be dependent on this value may cause issues if the header value is updated/corrected. I think we should either:

a. ensure that this header value is updated to reflect the actual minimum chunk size
b. set the `chunk_size` value as a const

cc @jbabichjapan  what do you think?

Reply:
Thanks, @tushdante!

`x-ton-min-chunk-size` is *only* returned on the POST request to the /1.1/ton/bucket/ta_partner endpoint. We only see that value once during the upload process. Does that change your thought on this?

Reply:
The code should in best scenario have elegant handling for network errors on the first attempt to upload or subsequent chunks, but otherwise think it should be okay to iterate after this check in